docker-compose.yml

```diff
@@ -73,7 +74,8 @@ services:
       - mysql_data:/var/lib/mysql
     ports:
       - '3306:3306'
-
+    cap_add:
+      - SYS_NICE  # CAP_SYS_NICE
```

Comment:
This will grant the container the CAP_SYS_NICE capability, which allows the container to raise process nice values, set real-time scheduling policies, set CPU affinity, and other operations.